lib/validation.js

```diff
@@ -122,10 +135,30 @@ function serialize (context, data, statusCode) {
   return JSON.stringify(data)
 }
 
+/**
+ * This function is called whenever an instance of `Error`
+ * must be serialized
+ */
+function serializeError (context, data, statusCode) {
```

Comment:
I don't like the fact that we have `statusCode` duplicated between `data` and `statusCode`.